fdbclient/BlobCipher.cpp

```diff
@@ -754,7 +754,6 @@ std::vector<Reference<BlobCipherKey>> BlobCipherKeyCache::getAllCiphers(const En
 	return keyIdCache->getAllCipherKeys();
 }
 
-namespace {
 int getEncryptAlgoHeaderVersion(const EncryptAuthTokenMode mode, const EncryptAuthTokenAlgo algo) {
```

Comment:
nit: probably rename to `getEncryptCurrentAlgoHeaderVersion` (should have named the same while adding it, my bad); especially as it look at the KNOB and gives the latest/current value.